package.json

```diff
@@ -38,6 +38,7 @@
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-react": "^7.4.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
```

Comment:
could we use [eslint-plugin-foreman](https://github.com/theforeman/foreman-js/tree/master/packages/eslint-plugin-foreman) which already has this and some of the other packages, as well as some basic rules config? 
If there are rules missing there that katello uses it would be great to add there and try to unify the linting as much as possible across the whole ecosystem.

Reply:
@tbrisker This sounds fine, however I tried adding `eslint-plugin-foreman` and ran `npx eslint webpack/`.  I ran into the following:

1. It still complained about not being able to find `eslint-plugin-react-hooks`.  So I added that back in to the package.json
2. Now `eslint` complains about not being able to find module `prettier`. I see that Foreman's package.json requires `prettier`, so I also add it to Katello.  Now I get:

```
✖ 3534 problems (3534 errors, 0 warnings)
  3531 errors and 0 warnings potentially fixable with the `--fix` option.
```

Considering the huge number of changes that `prettier` will require in Katello, I think it's better to do it this way:

1. Keep `eslint-plugin-react-hooks` here
2. In a separate PR, include both the addition of `eslint-plugin-foreman` and `prettier` (cc @Andrewgdewar).   Thoughts?

Reply:
side note: I also noticed that the version of eslint-plugin-react-hooks used in `eslint-plugin-foreman` is out of date:
```
"eslint-plugin-react-hooks": "^2.1.1"
```

Reply:
Sounds like a plan.